RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -38,3 +38,5 @@
                             ConvStep*
                             conversionStepTracks
                             )
+_iterTracking_Phase2PU140 = iterTracking.copyAndExclude([InitialStepPreSplitting])
+eras.trackingPhase2PU140.toReplaceWith(iterTracking, _iterTracking_Phase2PU140)
```

Comment:
Ah, I see now that the insertion of `recopixelvertexing` to reco sequence is still done with customize. Then the change above is fine (for now).